Example/Example/Common/BankCard/BankCardViewController.swift

```diff
@@ -14,6 +14,50 @@
 
 import TextInputKit
 
+struct BankCard {
+
+    var cardNumber: BankCardNumber? = nil
+
+    var expiryDate: BankCardExpiryDate? = nil
+
+    var cardHolderName: String = ""
+
+    var securityCode: String = ""
+
+}
+
+extension BankCard {
+
+    var prettyDescription: String {
+        let cardNumberDescription: String = {
```

Comment:
JFYI: It may be useful to create array of string entries line ("var: value") and then join it using comma as separator. It helps when in some cases var should not be presented in result string